.gitmodules

```diff
@@ -1,13 +1,14 @@
-[submodule "contracts/lib/forge-std"]
-	path = contracts/lib/forge-std
-	url = https://github.com/foundry-rs/forge-std
 [submodule "lib/forge-std"]
-	branch = v1.3.0
-[submodule "contracts/lib/openzeppelin-contracts"]
-	path = contracts/lib/openzeppelin-contracts
-	url = https://github.com/openzeppelin/openzeppelin-contracts
+	branch = v1
+	path = lib/forge-std
+	url = https://github.com/foundry-rs/forge-std
+	ignore = dirty
 [submodule "lib/openzeppelin-contracts"]
 	branch = v4.8.1
-[submodule "contracts/lib/subnet-evm"]
-	path = contracts/lib/subnet-evm
+	path = lib/openzeppelin-contracts
+	url = https://github.com/openzeppelin/openzeppelin-contracts
+	ignore = dirty
```

Comment:
Did you mean to change the `lib/forge-std` branch and/or add the `ignore = dirty` flags here? Seems like just changing the paths would be sufficient.

Reply:
When putting together this change, I noticed a few things:
1. our forge-std submodule was using a different branch than openzeppelin-contract's submodule
2. We weren't specifying a path to our forge-std submodule in remappings.txt

From what I could tell, this means that the exact forge-std version that's used is whatever one is found first when resolving include paths. To get around this, I added forge-std to remappings.txt explicitly, and updated our submodule to use the same version as openzeppelin-contracts

As for `ignore = dirty`, that is added as a developer convenience. I noticed that when playing with submodules it was quite easy to get to a state in which no submodule changes were made, but it was still showing as modified in `git status`. `ignore = dirty` shouldn't impact CI or fresh clones.